### DISCUSSION TITLE
pretty print?

### DISCUSSION THREAD

**Discussion by rex-remind101** [SeaQL/sea-query#263] [Q&A] [answered]
hello, we use `SelectStatement::to_string` in order to print queries in unit tests to confirm/deny any changes to a query that may occur when someone changes code. The diffs from query changes are difficult to parse however because the entire query is printed on 1 line when using `to_string`. Is there a mechanism for pretty printing the SQL so we can get cleaner diffs? I checked the docs and didn't seem to find anything there.

Thank you

**Comment by billy1624** [comment, marked as answer]
Hey @rex-remind101, I found this difficult when writing test cases. I use https://github.com/colin-kiegel/rust-pretty-assertions to assert the `SelectStatement::to_string`.
